src/image/cache/minio-s3/index.ts

```diff
@@ -0,0 +1,60 @@
+import ImageCache from '../abstract'
+import { createHash } from 'crypto'
+import { Client } from 'minio'
+
+export default class MinioS3ImageCache extends ImageCache {
+  private static client: Client
+
+  public constructor (config, req) {
+    super(config, req)
+    if (MinioS3ImageCache.client === undefined) {
+      MinioS3ImageCache.client = new Client(this.moduleConfig.libraryOptions)
+    }
+  }
+
+  public get moduleConfig (): any {
+    return this.config.imageable.caching[`minio-s3`]
```

Comment:
I would recommend to change it to `'` because we are not using template literal here - so no need for `.